src/xenia/kernel/xbdm/xbdm_misc.cc

```diff
@@ -465,6 +477,51 @@ dword_result_t DmSetDumpMode_entry(dword_t dump_mode) {
 }
 DECLARE_XBDM_EXPORT1(DmSetDumpMode, kDebug, kStub);
 
+struct X_DM_QUERY_MEMORY_STATISTICS_RESULT {
+  xe::be<uint32_t> size;
+  xe::be<uint32_t> total_pages;
+  xe::be<uint32_t> available_pages;
+  xe::be<uint32_t> stack_pages;
+  xe::be<uint32_t> virtual_page_table_pages;
+  xe::be<uint32_t> system_page_table_pages;
+  xe::be<uint32_t> pool_pages;
+  xe::be<uint32_t> virtual_mapped_pages;
+  xe::be<uint32_t> image_pages;
+  xe::be<uint32_t> file_cache_pages;
+  xe::be<uint32_t> contiguous_pages;
+  xe::be<uint32_t> debugger_pages;
+};
+static_assert_size(X_DM_QUERY_MEMORY_STATISTICS_RESULT, 48);
+
+dword_result_t DmQueryTitleMemoryStatistics_entry(
+    pointer_t<X_DM_QUERY_MEMORY_STATISTICS_RESULT> stats_ptr) {
+  if (stats_ptr->size != 48) return X_E_INVALIDARG;
+
+  xe::kernel::xboxkrnl::X_MM_QUERY_STATISTICS_RESULT stats;
+  stats.size = 104;
+  dword_result_t result = xe::kernel::xboxkrnl::MmQueryStatistics_entry(&stats);
+  if (result < 0) return result | 0x10000000;  // HRESULT_FROM_NT(result)
```

Comment:
This mask is kinda strange. If it is some base status mask then it should be defined somewhere